backend/controller/dal/async_calls.go

```diff
@@ -114,9 +124,27 @@ func (d *DAL) CompleteAsyncCall(ctx context.Context, call *AsyncCall, result eit
 		}
 
 	case either.Right[[]byte, string]: // Failure message.
-		_, err = tx.db.FailAsyncCall(ctx, result.Get(), call.ID)
-		if err != nil {
-			return translatePGError(err)
+		if call.RemainingAttempts > 0 {
+			nextBackoff := call.Backoff * 2
+			if nextBackoff > call.MaxBackoff {
+				nextBackoff = call.MaxBackoff
+			}
```

Comment:
```suggestion
			nextBackoff := min(call.Backoff * 2, call.MaxBackoff)
```

Reply:
copilot told me there was no such built in thing in go 😤